routes/weather.ts

```diff
@@ -281,27 +278,30 @@ export const getWateringData = async function( req: express.Request, res: expres
 		}
 	}
 
-	let scale: number = calculateWeatherScale( adjustmentMethod, adjustmentOptions, wateringData ),
-		rainDelay: number = -1;
+	let scale = -1,	rainDelay = -1;
+
+	if ( adjustmentMethod === ADJUSTMENT_METHOD.ZIMMERMAN ) {
+		scale = calculateZimmermanWateringScale( adjustmentOptions, wateringData );
+	}
 
 	if (wateringData) {
 		// Check for any user-set restrictions and change the scale to 0 if the criteria is met
 		if (checkWeatherRestriction(req.params[0], wateringData)) {
```

Comment:
Can we refactor `checkWeatherRestriction()` to be `checkCaliforniaRestrictions()` in the same way as you did for `calculateZimmermanScale()`. I.e. define, say, `californiaCheck = ( adjustmentValue >> 7 ) & 1` above and then only call function if true. I think it is much cleaner.

Reply:
I'm going to be doing some refactoring around this area of code in a near-future PR, and I'll clean up restriction handling there.

Reply:
Resolving through Issue #48